metriq_gym/quantinuum/provider.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from typing import Any
+
+from qbraid.runtime import QuantumProvider
+
+from .device import QuantinuumDevice, SUPPORTED_EMULATORS
+
+
+class QuantinuumProvider(QuantumProvider):
+    """qBraid provider for Quantinuum NEXUS emulator devices.
+
+    This implementation advertises H1-1E and H1-2E.
```

Comment:
See comment above about emulators vs. real devices. Imo we should implement this with support for both off the bat, and start testing it with syntax checkers, which are free, and emulators, which are cheaper.